static/app/components/replays/replayController.tsx

```diff
@@ -99,7 +99,7 @@ function ReplayPlayPauseBar({isCompact}: {isCompact: boolean}) {
               setCurrentTime(relativeTimeInMs(next.timestamp, startTimestampSec));
             }
           }}
-          aria-label={t('Jump to next event')}
+          aria-label={t('Fast-forward to next breadcrumb')}
```

Comment:
Should this be `Next breadcrumb` like the `title` text above?

Reply:
Same comment as above